### PR TITLE
Change signal names in Validate UN topology sample

### PR DESCRIPTION
**Pull request by har13205**
# Description

This PR is to change signal names in `Validate Utility Network topology` sample

## Type of change

- [x] Bug fix
- [ ] New sample implementation
- [ ] Sample viewer enhancement
- [ ] Other enhancement

**Platforms tested on**:

- [ ] Windows
- [ ] Android
- [ ] Linux
- [x] macOS
- [ ] iOS

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/UtilityNetwork/ValidateUtilityNetworkTopology/ValidateUtilityNetworkTopology.h [hunk 1]
```diff
@@ -46,13 +46,13 @@ class ValidateUtilityNetworkTopology : public QObject
...
-  Q_PROPERTY(bool isClearBtnEnabled MEMBER m_isClearBtnEnabled NOTIFY isClearBtnEnabled)
-  Q_PROPERTY(bool busy MEMBER m_busy NOTIFY isBusy)
-  Q_PROPERTY(bool isStateBtnEnabled MEMBER m_isStateBtnEnabled NOTIFY isStateBtnEnabled)
+  Q_PROPERTY(bool isUpdateWindowVisible MEMBER m_isUpdateWindowVisible NOTIFY updateWindowVisibilityChanged)
+  Q_PROPERTY(QString updateFieldName MEMBER m_updateFieldName NOTIFY fieldNameChanged)
+  Q_PROPERTY(bool isValidateBtnEnabled MEMBER m_isValidateBtnEnabled NOTIFY validateBtnStateChanged)
+  Q_PROPERTY(bool isTraceBtnEnabled MEMBER m_isTraceBtnEnabled NOTIFY traceBtnStateChanged)
+  Q_PROPERTY(bool isClearBtnEnabled MEMBER m_isClearBtnEnabled NOTIFY clearBtnStateChanged)
+  Q_PROPERTY(bool busy MEMBER m_busy NOTIFY busyStateChanged)
+  Q_PROPERTY(bool isStateBtnEnabled MEMBER m_isStateBtnEnabled NOTIFY stateBtnStateChanged)
```
Comment (JamesMBallard) [inline ArcGISRuntimeSDKQt_CppSamples/UtilityNetwork/ValidateUtilityNetworkTopology/ValidateUtilityNetworkTopology.h:55]: property names and signals should follow the naming convention of `property` and `propertyChanged` throughout. Please review.

**Comment by GuillaumeBelz** [comment]
I'm not a big fan of abbreviations, like `btn` instead of `button`. I think this is not necessary to save 3 chars and this is less explicit.

File: ArcGISRuntimeSDKQt_CppSamples/UtilityNetwork/ValidateUtilityNetworkTopology/ValidateUtilityNetworkTopology.h [hunk 2]
```diff
@@ -104,14 +104,14 @@ class ValidateUtilityNetworkTopology : public QObject
...
+  QString m_fieldName;
   QList<QString> m_choices;
-  bool m_isUpdateWindowVisible = false;
-  bool m_isValidateBtnEnabled = false;;
-  bool m_isTraceBtnEnabled = false;
-  bool m_busy = false;
-  bool m_isClearBtnEnabled = false;
-  bool m_isStateBtnEnabled = false;
+  bool m_updateWindowVisibility = false;
+  bool m_validateButtonAvailability = false;;
```
Comment (JamesMBallard) [inline ArcGISRuntimeSDKQt_CppSamples/UtilityNetwork/ValidateUtilityNetworkTopology/ValidateUtilityNetworkTopology.h:110]:
```suggestion
  bool m_validateButtonAvailability = false;
```

**Review by JamesMBallard — APPROVED** [review]
One small thing. No need for another review.

**Pull request merged**